packages/eslint-plugin/scripts/codegen.ts

```diff
@@ -86,97 +91,39 @@ const conditional = (cond: string, content?: boolean | string) => (content ? con
 const link = (content: string, url?: string) => (url ? `<a href="${url}">${content}</a>` : content)
 
 async function generateReadme(rules: FoundRule[]) {
-  const code = `
-# @react-three/eslint-plugin
-
-[![Version](https://img.shields.io/npm/v/@react-three/eslint-plugin?style=flat&colorA=000000&colorB=000000)](https://npmjs.com/package/@react-three/eslint-plugin)
-[![Twitter](https://img.shields.io/twitter/follow/pmndrs?label=%40pmndrs&style=flat&colorA=000000&colorB=000000&logo=twitter&logoColor=000000)](https://twitter.com/pmndrs)
-[![Discord](https://img.shields.io/discord/740090768164651008?style=flat&colorA=000000&colorB=000000&label=discord&logo=discord&logoColor=000000)](https://discord.gg/ZZjjNvJ)
-[![Open Collective](https://img.shields.io/opencollective/all/react-three-fiber?style=flat&colorA=000000&colorB=000000)](https://opencollective.com/react-three-fiber)
-[![ETH](https://img.shields.io/badge/ETH-f5f5f5?style=flat&colorA=000000&colorB=000000)](https://blockchain.com/eth/address/0x6E3f79Ea1d0dcedeb33D3fC6c34d2B1f156F2682)
-[![BTC](https://img.shields.io/badge/BTC-f5f5f5?style=flat&colorA=000000&colorB=000000)](https://blockchain.com/btc/address/36fuguTPxGCNnYZSRdgdh6Ea94brCAjMbH)
-
-An ESLint plugin which provides lint rules for [@react-three/fiber](https://github.com/pmndrs/react-three-fiber).
-
-## Installation
-
-\`\`\`bash
-npm install @react-three/eslint-plugin --save-dev
-\`\`\`
-
-## Configuration
-
-Use the recommended [config](#recommended) to get reasonable defaults:
-
-\`\`\`json
-"extends": [
-  "plugin:@react-three/recommended"
-]
-\`\`\`
-
-If you do not use a config you will need to specify individual rules and add extra configuration.
-
-Add "@react-three" to the plugins section.
-
-\`\`\`json
-"plugins": [
-  "@react-three"
-]
-\`\`\`
-
-Enable the rules that you would like to use.
+  const filepath = join(srcDir, '../', 'README.md')
+  const readme = await fs.readFile(filepath, 'utf-8')
 
-\`\`\`json
-"rules": {
-  "@react-three/no-clone-in-frame-loop": "error"
-}
-\`\`\`
+  const rows: string[] = []
 
-## Rules
+  for (const rule of rules) {
+    const docsPath = await ruleDocsPath(rule.moduleName)
+    const row = `| ${link(rule.moduleName, docsPath)} | ${rule.module.meta?.docs?.description} | ${conditional(
```

Comment:
Rule names now have a link (as rule docs path enforces there to always be a doc path!)